discussions/api.py

```diff
@@ -267,46 +265,58 @@ def remove_from_channel(channel_name, discussion_username):
     remove_contributor_from_channel(channel_name, discussion_username)
 
 
-def sync_user_to_channels(user_id):
+def sync_channel_memberships():
```

Comment:
Since this runs every minute, are we doing anything to prevent two instances of this task from running concurrently?

Reply:
Not right now, we don't have a good locking utility, although code exists in dashboard/tasks.py, that would add time to generalize. An alternative would be to space out the task runs.